### PR TITLE
This PR is for debugging the lint issues.

### PR DESCRIPTION
**Pull request by anubhavsinha98**
This PR fixes the ```htmllint``` issue and fix the wrong-import-order.

### CODE REVIEW

File: app/core/domain/email_manager.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 """Config functions for managing email notifications."""
-
+# Check.
```
Comment (seanlip) [inline app/core/domain/email_manager.py]: Is this intended?

Comment (anubhavsinha98) [reply app/core/domain/email_manager.py]: That was just for testing I will delete it.

**Comment by anubhavsinha98** [comment]
Hi @seanlip, below is the error message which was generated by the htmllint when we were using the social word.

Screenshot from 2019-04-17 12-36-19

**Comment by seanlip** [comment]
Ah I see, thanks for explaining. Strange rule, but I think your fix is fine. Thanks!

File: app/core/domain/email_manager.py (hunk 1 above)
Comment (seanlip) [reply app/core/domain/email_manager.py]: Thanks -- once you fix this we can merge it!

Comment (anubhavsinha98) [reply app/core/domain/email_manager.py]: Done @seanlip

**Pull request merged**